app/templates/join.hbs

```diff
@@ -1,2 +1,14 @@
 {{page-title 'Join'}}
-<Stepper />
\ No newline at end of file
+{{#if this.login.isLoading}}
+<div data-test-loading class='intro__loading'>
+    <Fa-Icon @size='2x' @icon='circle-notch' @spin={{true}} />
+</div>
+{{else}}
+{{#if this.login.userData.roles.archived}}
+<Card>
+<h3 class="archived__heading">Archived User</h3>
```

Comment:
Why not create seperate component

Reply:
I have created a separate component for the card but for archived it is a very small description so it depends to create separate or not